<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CArmScanSetting</class>
 <widget class="QWidget" name="CArmScanSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>328</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CArmScanSetting</string>
  </property>
  <widget class="QWidget" name="captureModeWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>328</width>
     <height>160</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="KVp_mAWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>161</y>
     <width>328</width>
     <height>117</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="hardwareControlWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>279</y>
     <width>328</width>
     <height>344</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>161</y>
     <width>328</width>
     <height>1</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>1</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>1</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgba(255,255,255,0.1);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>279</y>
     <width>328</width>
     <height>1</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>1</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>1</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgba(255,255,255,0.1);</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
